<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Assets/eye.png</normaloff>../Assets/eye.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>240</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_12" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QWidget" name="widget_14" native="true">
            <property name="maximumSize">
             <size>
              <width>800</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6"/>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_11" native="true">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QFrame" name="frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QWidget" name="widget_8" native="true"/>
          </item>
          <item>
           <widget class="QLabel" name="icon_profile">
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../IMG/user.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_9" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_10" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>550</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6"/>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>90</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLineEdit" name="txt_Username">
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>48</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>JetBrains Mono</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    border: 2px solid #0078D7;
	border-radius: 10px;
	padding: 8px 12px;
    background-color: #F4F6FF;
    color: #333;
	box-shadow: 0px 4px 10px rgba(0, 120, 215, 0.2);
}

</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="placeholderText">
                <string>Username</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>90</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="txt_Password">
               <property name="maximumSize">
                <size>
                 <width>310</width>
                 <height>48</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>JetBrains Mono</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    border: 2px solid #0078D7;
	border-radius: 10px;
	padding: 8px 12px;
    background-color: #F4F6FF;
    color: #333;
	box-shadow: 0px 4px 10px rgba(0, 120, 215, 0.2);
}</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="placeholderText">
                <string>Password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="show_password_btn">
               <property name="styleSheet">
                <string notr="true">QPushButton {
    border: 2px solid #0078D7;
    border-radius: 6px;
    background-color: white;
    padding: 6px 12px;
    color: #0078D7;
    font-weight: bold;
	box-shadow: 0px 4px 10px rgba(0, 120, 215, 0.2);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_13" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="btn_LogIn">
               <property name="styleSheet">
                <string notr="true">QPushButton {
    border: 2px solid #0078D7;
    border-radius: 10px;
    padding: 6px 12px;
    background-color: #0078D7;
    color: white;
    font-weight: bold;
}</string>
               </property>
               <property name="text">
                <string>Login</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_15" native="true">
            <property name="maximumSize">
             <size>
              <width>290</width>
              <height>50</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>JetBrains Mono</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Need an account?</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_SignUpnow">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>JetBrains Mono</family>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color:#0078D7;
border-radius:10px;
border: 1px solid #ccc;
padding:2px 2px;</string>
               </property>
               <property name="text">
                <string>SignUp now</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
